permissions_validators/src/public_blockchain/burn.rs

```diff
@@ -213,18 +219,19 @@ impl HasToken for GrantedByAssetOwner {
 
 /// Validator that checks Grant instruction so that the access is granted to the assets
 /// of the signer account.
-#[derive(Debug, Copy, Clone, Serialize)]
+#[derive(Debug, Display, Copy, Clone, Serialize)]
+#[display(fmt = "the signer is the asset owner")]
```

Comment:
Initially it was done by purpose. It's implicit, but all this messages for types implementing `IsGrantAllowed` (or `IsRevokeAllowed`) will be printed with the message from `IsGrantAllowedAsValidator` (or `IsRevokeAllowedAsValidator`). So this will look like this:

```
Allow to grant `CanBurnUserAssets` permission token if `the signer is the asset owner`
```

Reply:
So inside quotes the *variables* are placed

Reply:
Or you think it still will be better to start with capital letter?

Reply:
Also, take a look at examples I've provided in the PR description